torch/nn/quantized/modules/conv.py

```diff
@@ -56,6 +58,8 @@ def extra_repr(self):
             s += ', padding={padding}'
         if self.dilation != (1,) * len(self.dilation):
             s += ', dilation={dilation}'
+        if self.output_padding != (0,) * len(self.output_padding):
```

Comment:
what about `transposed`?

Reply:
I don't think we need it in the `str` because the name of the module already has transpose in it.